enterprise/frontend/src/metabase-enterprise/metabot/reactions/queries.ts

```diff
@@ -0,0 +1,31 @@
+import { updateQuestion } from "metabase/query_builder/actions";
+import { getQuestion } from "metabase/query_builder/selectors";
+import * as Lib from "metabase-lib";
+import type { MetabotSortQueryReaction } from "metabase-types/api";
+
+import type { ReactionHandler } from "./types";
+
+export const sortQuery: ReactionHandler<MetabotSortQueryReaction> =
+  reaction =>
+  async ({ dispatch, getState }) => {
+    const question = getQuestion(getState());
+    if (!question) {
+      return;
```

Comment:
We need to tell the user something is wrong and halt any further processing of reactions. You can see an example in the visualization reactions file:
```
      if (!question) {
        dispatch(
          stopProcessingAndNotify(
            t`You have to be viewing a question for me to modify it.`,
          ),
        );
        return;
      }
``` 
If you rebase, you can call that without any message to get a generic "I can’t do that, unfortunately." message shown to the user.


I will add docs on this.